Scripts/files.py

```diff
@@ -296,10 +299,71 @@ def check_for_update(currentVersion, updateReleaseChannel, silentCheck=False):
             print(f" > And don't forget to report any problems you encounter here: {F.YELLOW}TJoe.io/bug-report{S.R}")
           input("\nPress Enter to Exit...")
           sys.exit()
+        elif platform.system() == "Linux":
+          # Current working directory
+          cwd = os.getcwd()
+          # what we want the tar file to be called on the system
+          tarFileName = "yt-spammer.tar.gz"
+          # Name of this file
+          # Temp folder for update
+          stagingFolder = "temp"
+
+          # Fetch the latest update
+          print(f"\n> Downloading version: {F.GREEN}{latestVersion}{S.R}")
+
+          url = f'https://codeload.github.com/ThioJoe/YT-Spammer-Purge/tar.gz/refs/tags/v{latestVersion}'
+          r = requests.get(url, stream=True)
+          if(r.status_code == 200):
```

Comment:
```suggestion
          if r.status_code == 200:
```
The code above don't have parentheses, removing them should be fine